src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import './utils/variables';
+@import './blocks/body';
+@import './blocks/nav';
+@import './blocks/card';
+@import './blocks/header';
+@import './blocks/main';
+@import './blocks/stars';
+@import './blocks/stars';
```

Comment:
The import for './blocks/stars' is duplicated. Remove the duplicate import to avoid redundancy.